### DISCUSSION TITLE
Apple airport problem.

### DISCUSSION THREAD

**Discussion by nickforev** [RavenSystem/esp-homekit-devices#1716] [Q&A]
Hello. After connecting devices flashed with HAA 10.1.1, after a while time (30min) it disappears from the Homekit application.
In the Airport router, the connection is active, you can go into the setup mode.
Apple TV as Home Center.
This is not the first time I have encountered a similar problem when connecting Apple Airport + HAA
Tell me how to fix?
![image](https://user-images.githubusercontent.com/67782540/151972946-fa06a95b-f5fd-46a5-a128-eb0d4dd13cde.png)
![image](https://user-images.githubusercontent.com/67782540/151973125-265aaf7a-8329-4504-a388-0c0a6eb6fc34.png)

**Comment by VaBanck** [comment]
Может быть проблема в самой esp. У меня тоже есть одна такая, работает но через какое-то время вырубается

**Reply by nickforev** [reply]
Я проверил на нескольких basic r3, sa-018, mini tiger. На некоторых форумах читал что проблема связана с MTU и mdns. Возможно спецы знают куда смотреть?

**Reply by VaBanck** [reply]
Зайди в чат discord, там быстрее получишь ответ

**Reply by VaBanck** [reply]
MTU у меня 1500

**Reply by nickforev** [reply]
> Зайди в чат discord, там быстрее получишь ответ 

Link pls.

**Reply by VaBanck** [reply]
https://discord.com/channels/594630635696553994/913814068803031080/938051239982268456

**Reply by nickforev** [reply]
> https://discord.com/channels/594630635696553994/913814068803031080/938051239982268456

![image](https://user-images.githubusercontent.com/67782540/152026106-5f2b5ed5-22f4-4e57-b50d-838072b5aa05.png)

**Reply by VaBanck** [reply]
https://discord.gg/v8hyxj2
